src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx

```diff
@@ -295,9 +295,9 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
             subject: 'jurisdiction_id',
           },
           {
-            comparator: 'Case Confirmation',
+            comparator: 'Case_Confirmation',
             operator: '==',
-            subject: 'action_code',
+            subject: 'goal_id',
```

Comment:
Why the change from action_code to goal_id?

Reply:
This are the columns present `task_identifier, plan_id, jurisdiction_id, goal_id, geojson` i think it was wrong to use `action_code`

Reply:
`action_code` is in geojson > properties > action_code